File: testdata/bench/rust/BenchmarkTest00090.rs
use actix_web::{web, HttpResponse};

// Safe: Hardcoded HTML, no user input
async fn handler(_query: web::Query<()>) -> HttpResponse {
    let html = "<html><body>Hello World</body></html>";
    HttpResponse::Ok().content_type("text/html").body(html)
}

File: testdata/bench/cpp/BenchmarkTest00090.cpp
#include <iostream>
#include <cstdlib>
#include <string>

int main() {
    char *comment = getenv("QUERY_STRING");
    std::string html = "<html><body><div>" + std::string(comment ? comment : "") + "</div></body></html>";
    std::cout << "Content-Type: text/html\r\n\r\n";
    std::cout << html;
    return 0;
}
